react-visualizer/src/App.js

```diff
@@ -167,6 +167,8 @@ function Main() {
     { value: "coins19.json", label: "coins19" },
     { value: "coins20.json", label: "coins20" },
     { value: "coins21.json", label: "coins21" },
+    { value: "coins23.json", label: "coins23" },
```

Comment:
元がvalueとか分かりにくくて申し訳ないです…:pray:
```suggestion
    { value: "coins23-24.json", label: "coins23" },
```